tools/pkglint/lib/rules.ts

```diff
@@ -314,6 +324,50 @@ export class StabilitySetting extends ValidationRule {
   }
 }
 
+export class FeatureStabilityRule extends ValidationRule {
+  public readonly name = 'package-info/feature-stability';
+
+  public validate(pkg: PackageJson): void {
+    if (pkg.json.private || !pkg.json.featureStability) {
+      return;
+    }
+
+    const partTable = pkg.json.featureStability.map((feature: any) => `| ${feature.name} | ${feature.stability} |`);
+    const stabilityBanner: string = [
+      '<!--BEGIN STABILITY BANNER-->',
+      '---',
+      '',
+      '| Features | Stability |',
+      '| --- | --- |',
+      '| CFN Resources | Stable |',
```

Comment:
What happens for modules that do not have CFN resources? I think we can auto-deduce this from package.json and omit in this case